README.md

```diff
@@ -10,6 +10,8 @@ The repository includes:
 4. **nuage-ospd13-dockerfiles**: (Deprecated : This is only required for 5.4.1u1 to 5.4.1u3). Starting from 5.4.1.U4, please refer to Red Hat Build Services for acquiring Nuage containers.
 5. **Documentation**: The details of integration, architecture and deployment steps can be found under Documentation folders, as following.
 
+    * For OSP Director 13 with + Nuage release 6.0.1: [README.rst](Documentation/6.0.1/README.rst)
+
     * For OSP Director 13 with + Nuage release 5.4.1.U4: [README.rst](Documentation/5.4.1/README.rst)
```

Comment:
For OSP Director 13 with + Nuage release 5.4.1.U4 and later

Reply:
ok, done